src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private Color color;
+    private int side;
+    private int height;
+
+    public RightTriangle(Color color, int side, int height) {
+        this.color = color;
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return (side * height) / 2;
```

Comment:
integer division in floating-point context .
Please pay attention to [this](https://www.javatpoint.com/java-int-to-double) article.

Reply:
Should I give height field a double value by default or should I cast it to double in a getArea method? Hope I changed it in a right way